int32 sliceId
int32 width
int32 height
treeD_client_pkg/PointMsg[] points
